spec/lib/vanagon/component/source_spec.rb

```diff
@@ -108,7 +108,7 @@
           'http://buildsources.delivery.puppetlabs.net'
       end
       it "applies rewrite rules to HTTP URLs" do
-        expect(klass.source(original_http_url, sum: sum, workdir: workdir).url)
+        expect(klass.source(original_http_url, sum: sum, workdir: workdir, sum_type: "md5").url)
           .to eq(rewritten_http_url)
       end
     end
```

Comment:
Can we see some tests around the new SHA sum methods? Even just validating that things work as expected would make me 😄 .